rl_chain/rl_chain_base.py

```diff
@@ -199,55 +201,53 @@ def _learn(self, vw_ex):
         self.workspace.learn_one(multi_ex)
 
 
+def embed_string_type(item: Union[str, _Embed], model: Any, default_namespace: Optional[str] = None) -> Dict[str, str]:
```

Comment:
looks like default_namespace is not needed anymore?

Reply:
sorry, it is still used, but it is not "default" anymore

Reply:
maybe should be renamed to `namespace`

Reply:
yeah, there is no alternative to it anyway

Reply:
kay done